zoo-css-challenge/index.html

```diff
@@ -80,52 +83,72 @@ <h3>Height and Size</h3>
 
 		<section class="news">
 			<h2>News</h2>
-			<div class="card">
+			<div class="container">
+			<div class="box">
+				
 				<time>2010-04-16</time>
-				<h4>Helping wild turtles in Seychelles</h4>
+				<h3>Helping wild turtles in Seychelles</h3>
 				<a>Find out More</a>
+				<br>
 			</div>
-			<div class="card">
+			
+			<div class="box">
 				<time>2010-04-12</time>
 				<h4>Learning about the rainforest</h4>
-				<a>Find out More</a>
+				<a>Find out More</a><br>
+			</div>
+			<div class="box">
+				<time>2021-06-04</time>
+				<h5>‘Which big cat are you?’</h5>
+				<a>Find out More</a><br>
+			</div>
 			</div>
 		</section>
 
 		<section class="education">
-			<div class="education-container">
-				<h3>Learning resources for schools</h3>
+			<div class="container">
+				<h1>Learning resources for schools</h1>
 				<p>Explore our activities, videos, and lesson plans to support a wide range of curriculum topics</p>
-				<button>Find them here</button>
+				<button id= "Find them here">Find them here</button>
```

Comment:
```suggestion
				<button id= "find-them-here">Find them here</button>
```
IDs and classes can't have spaces in them. Instead, normally we use dashes instead of spaces and make the text lowercase